eng/pipelines/templates/variables/globals.yml

```diff
@@ -8,7 +8,7 @@ variables:
   # Maven build/test options
   MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
   # See https://github.com/actions/virtual-environments/issues/1499 for more info about the wagon options
-  WagonOptions: '-Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120'
+  WagonOptions: '-Dmaven.wagon.httpconnectionManager.ttlSeconds=120'
```

Comment:
Did you intend to include this as part of this PR?

Reply:
Was checking CI performance what this is removed but it appears the cache resolved itself in a previous run so no chance of a timeout was left. Removing it in the next commit

Reply:
If you want to do some testing without the cache then you set manually queue the build and set the CacheSalt to some value which will invalidate the cache for that run. That is what the CacheSalt is for.